compiler/rustc_ast_passes/src/feature_gate.rs

```diff
@@ -152,8 +152,7 @@ impl<'a> PostExpansionVisitor<'a> {
     }
 
     fn check_late_bound_lifetime_defs(&self, params: &[ast::GenericParam]) {
-        // Check only lifetime parameters are present and that the lifetime
-        // parameters that are present have no bounds.
+        // Check parameters that are present have no bounds.
```

Comment:
```suggestion
        // Check parameters that are present have no bounds.
```

```suggestion
        // Check only lifetime parameters are present and that the generic parameters have no bounds.
```

We should keep the first comment since we're still making sure that only lifetime parameters are present.

Reply:
Done. Thank you.